libs/http-parser/Makefile

```diff
@@ -7,30 +7,27 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=libhttp-parser
```

Comment:
Please do not rename this package, some packages are already depending on libhttp-parser